backend/db/schema.rb

```diff
@@ -19,6 +19,7 @@
     t.string "wallet_address", default: "", null: false
     t.string "encrypted_password", default: "", null: false
     t.string "jti", null: false
+    t.string "signature", null: false
```

Comment:
No need to store `signature` into the database, since this often changes every time a nonce value gets generated

Reply:
Ahh okay thanks, Carlo will make change